Tools/Scripts/webkitdirs.pm

```diff
@@ -914,7 +914,7 @@ sub determineXcodeSDKPlatformName {
     if (checkForArgumentAndRemoveFromARGV("--maccatalyst")) {
         $xcodeSDKPlatformName = "maccatalyst";
     }
-    if (checkForArgumentAndRemoveFromARGVGettingValue("--sdk", \$sdk)) {
+    if (checkForArgumentAndRemoveFromARGVGettingValue("--sdk", \$sdk) || checkForArgumentAndRemoveFromARGVGettingValue("SDKROOT", \$sdk)) {
```

Comment:
Not sure if we should support `set-webkit-configuration SDKROOT=...` but that's how folks pass it to `make` so maybe it's reasonable here too? Also happy to revert this.

Reply:
No, `make` is different in how one passes arguments.

I don't think we need to support `set-webkit-configuration SDKROOT=blargh`.